Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -736,7 +736,7 @@ protected virtual void BeginDelay(UseDelayComponent? component = null)
         public bool AltInteract(EntityUid user, EntityUid target)
         {
             // Get list of alt-interact verbs
-            var verbs = _verbSystem.GetLocalVerbs(target, user, VerbType.Alternative)[VerbType.Alternative];
+            var verbs = _verbSystem.GetLocalVerbs(target, user, new() { typeof(AlternativeVerb) });
```

Comment:
this is kind of ass syntax as opposed to what it was before, but i think this is uncommon enough that i'm fine with it

Reply:
I added a wrapper function for single-type requests, so now its just
```cs
_verbSystem.GetLocalVerbs(target, user, typeof(AlternativeVerb))
```
Which makes it somewhat better.

Reply:
sgtm